frontend/src/metabase/entities/actions/actions.ts

```diff
@@ -94,14 +103,28 @@ const Actions = createEntity({
   nameOne: "action",
   path: "/api/action",
   api: {
-    create: (params: CreateQueryActionParams | CreateImplicitActionParams) =>
-      ActionsApi.create(params),
-    update: (params: UpdateQueryActionParams | UpdateImplicitActionParams) =>
-      ActionsApi.update(params),
+    create: (params: CreateActionParams) => ActionsApi.create(params),
+    update: (params: UpdateActionParams) => {
+      // Changing action type is not supported
+      const cleanParams = _.omit(params, "type");
+      return ActionsApi.update(cleanParams);
+    },
   },
   actions: {
     enableImplicitActionsForModel,
   },
+  writableProperties: [
```

Comment:
Currently, only 2 entities `actions` and `questions` seem to use `writableProperties`. When should we use this option?